src/core/client/admin/routes/configure/sections/auth/components/FacebookConfig.tsx

```diff
@@ -51,7 +51,7 @@ const FacebookConfig: FunctionComponent<Props> = ({
       <HorizontalGutter size="double">
         <Localized
           id="configure-auth-facebook-toEnableIntegration"
-          link={<FacebookLink />}
```

Comment:
Why the switch to `Link`?

Reply:
I wanted to stay consistent: It's a Component => It's uppercase.